lib/admin_credentials.rb

```diff
@@ -19,7 +19,21 @@ def initialize(env, **opts)
 
     # @return [APIAccessor]
     def get
-      res = master_host.exec_as(:admin, "cat /root/.kube/config", quiet: true)
+      res = master_host.exec_admin("cat /root/.kube/config", quiet: true)
+      if !res[:success] && res[:response].include?("No such file or directory")
+        # try to find kubeconfig in other locations
+        locations = [["kubeconfig", "/etc/kubernetes/static-pod-resources"]]
```

Comment:
Heard from @jianlinliu that Dev is [in-flight](https://github.com/openshift/installer/issues/862#issuecomment-451605549) making cluster not ssh-able. So I think we still need URL support for kubeconfig.

Reply:
This is unbelievable, how is customer supposed to access cluster if it is not SSH-able?

Reply:
I don't read the linked to issue that SSH will be disabled. It makes not sense IMO to disable SSH.